tools/clang/unittests/HLSLExec/LongVectors.cpp

```diff
@@ -1593,6 +1593,38 @@ template <typename T> T waveMultiPrefixProduct(T A, UINT) {
 
 template <typename T> struct Op<OpType::WaveMatch, T, 1> : StrictValidation {};
 
+// Helper struct to build the expected result for WaveMatch tests.
+struct Wave {
+  uint64_t LowWaveMask;
+  uint64_t HighWaveMask;
+
+  uint64_t LowBits;
+  uint64_t HighBits;
```

Comment:
We should aim to make these private.

What if there was function that looked like this?

`void SetExpected(std::vector<UINT>::iterator It)`

That would write the values into the 4 elements of the vector?